### PR TITLE
feat: add <Balance />

### PR DESCRIPTION
**Pull request by eragon512**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**
Unit test

### CODE REVIEW

File: src/components/Balance.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * Balance is a React component that renders the account balance for the given Ethereum address.
+ * It displays the user's balance, with the no. of decimals specified by the 'decimalDigits' prop.
+ * By default, 'decimalDigits' is set to 3.
+ *
+ * @param {Address} props.address - The Ethereum address for which to display the balance.
+ * @param {string} [className] - Optional CSS class for custom styling.
+ * @param {number} [decimalDigits=3] - Determines the no. of decimal digits to be displayed.
+ * @param {React.HTMLAttributes<HTMLSpanElement>} [props] - Additional HTML attributes for the span element.
+ */
+export function Balance({ address, className, decimalDigits = 3, ...props }: BalanceProps) {
```
Comment (Sneh1999) [inline src/components/Balance.tsx:36]: Can we add this functionality to the avatar is possible?

Comment (eragon512) [reply src/components/Balance.tsx:36]: not sure if that's the best approach here - generally, it's best to keep each component small and atomic, with a single, clear responsibility

adding balance data to the avatar might add multiple responsibilities inside a single component

if users need balance data in their avatar, having a separate component allows them to stitch the avatar and balance components together in the way they feel fits their use case best

LMK your thoughts on this

Comment (Zizzamia) [reply src/components/Balance.tsx:36]: This week is hard for us to ship a new component.

Let's table this till the end of ETH Denver.

But, I love we at least started the convo.

**Comment by Zizzamia** [comment]
Closing this for now, as `balance` is not activelly develop yet.

**Pull request closed without merging**